bittensor_cli/cli.py

```diff
@@ -2804,7 +2804,17 @@ def stake_get_children(
         wallet_path: Optional[str] = Options.wallet_path,
         network: Optional[str] = Options.network,
         chain: Optional[str] = Options.chain,
-        netuid: int = Options.netuid,
+        netuid: Optional[str] = typer.Option(
```

Comment:
This is not `netuid: int = Options.netuid` because if `--all` flag is entered we dont want to prompt the user for netuid. (set prompt to false)

Reply:
See comment below